### PR TITLE
chore(ExtensionBanner): adding lab extension to recommendation

### PR DESCRIPTION
**Pull request by axel7083**
### Screenshot / video of UI

See https://github.com/containers/podman-desktop-internal/issues/243#issuecomment-2076657881

### What issues does this PR fix or reference?

Fixes https://github.com/containers/podman-desktop-internal/issues/243

### How to test this PR?

How to test ?

Change the property `publishDate` in the `recommendations.json` file for the extension that you want to see.

> We can only see one, so put some very far date for the extension you want to hide.

- [ ] Tests are covering the bug fix or the new feature

### CODE REVIEW

File: featured.json [hunk 1]
```diff
@@ -14,6 +14,14 @@
...
+    },
+    {
+      "extensionId": "redhat.ai-lab",
+      "displayName": "AI-Lab",
```
Comment (benoitf) [inline featured.json:20]: Podman AI Lab

File: featured.json [hunk 2]
```diff
@@ -14,6 +14,14 @@
...
+      "shortDescription": "Try the new AI-Lab extension",
```
Comment (benoitf) [inline featured.json]:
```suggestion
      "shortDescription": "Try the new Podman AI Lab extension",
```

File: recommendations.json [hunk 3]
```diff
@@ -9,6 +9,17 @@
...
       }
+    },
+    {
+      "extensionId": "redhat.ai-lab",
+      "publishDate": "2024-05-06",
+      "title": "Supercharge your apps with AI Lab!",
```
Comment (benoitf) [inline recommendations.json]:
```suggestion
      "title": "Supercharge your apps with Podman AI Lab!",
```

File: featured.json (hunk 1 above)
Comment (benoitf) [inline featured.json]:
```suggestion
      "displayName": "Podman AI Lab",
```

**Review by benoitf — APPROVED** [review]

File: recommendations.json [hunk 4]
```diff
@@ -9,6 +9,17 @@
...
       }
+    },
+    {
+      "extensionId": "redhat.ai-lab",
+      "publishDate": "2024-05-06",
```
Comment (axel7083) [inline recommendations.json]:
```suggestion
      "publishDate": "2024-05-06T16:00:00.000Z",
```

See https://github.com/containers/podman-desktop-internal/issues/243#issuecomment-2077558289

File: recommendations.json [hunk 5]
```diff
@@ -9,6 +9,17 @@
...
       }
+    },
+    {
+      "extensionId": "redhat.ai-lab",
+      "publishDate": "2024-05-06T16:00:00.000Z",
```
Comment (axel7083) [inline recommendations.json]:
```suggestion
      "publishDate": "2024-05-07T14:25:00.000Z",
```

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**